packages/expo-dev-client/README.md

```diff
@@ -4,6 +4,10 @@ This is a preview version of the Expo Development Client.
 
 ## Documentation
 
+`expo-dev-client` is an npm package installable in any Expo or React Native project. Once installed, any Debug builds of your application will gain an extensible debug menu and the ability to load projects from Expo CLI. Release builds of your application will not change other than the addition of a few header files.
+
+See [Development Client documentation](https://docs.expo.io/clients/introduction/) for more information.
```

Comment:
Talking about "Development Client" is often unclear between the module or a development client for a project (and misses an opportunity to reinforce that it's the developer's development client, not expo's) so I've been avoiding saying Development Client / Dev Client in favor of expo-dev-client or (your / a) custom development client.

In this case I think the referent is clear so you can just say "See the documentation..."